.golangci.yml

```diff
@@ -2,6 +2,11 @@ run:
   deadline: 10m
   skip-files:
   - 'zz_generated\.(\w*)\.go$'
+  skip-dirs:
```

Comment:
Why are we skipping the linter in these directories now? Otherwise, lgtm–nice work, thanks for sticking with this @willie-yao!

Reply:
We had to skip them due to `v1alpha3` and `v1alpha4` being deprecated in CAPI, which fails the test CI job. This was added to suppress those deprecation warnings. I will be working on removing those API versions completely for the 1.9 release.